archetype-module-content/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<archetype-descriptor xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd"
+                      name="openmrs-archetype-module"
+                      xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0"
+                      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <requiredProperties>
+        <requiredProperty key="groupId">
+            <defaultValue>org.openmrs.module</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="artifactId" />
+        <requiredProperty key="version">
+            <defaultValue>1.0.0-SNAPSHOT</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="package">
+            <defaultValue>${groupId}.${artifactId}</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="moduleName" />
+        <requiredProperty key="moduleDescription"/>
+        <requiredProperty key="moduleAuthor">
+            <defaultValue>mherman22</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="moduleClassnamePrefix" />
```

Comment:
We don't need this for content packages, I don't think.

Reply:
True that! I will add a check that skips adding that property for **content-package** type.